.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+**Checklists** 
+- [ ] Added impacted library name to the issue description
+
+**Library(ies) impacted by this PR:**
+
+
+**Issue(s) associated with this PR:**
+
+
+**Describe the problem(s) that is addressed by this PR:**
+
+
+**Command used to generate this PR:**_(Applicable only to SDK release request PRs)_
```

Comment:
Any reason we would not need this for other packages we have where there is generated code + convenience layer? Or the RLC packages?

Reply:
For all the other packages, the standard command is `rushx build`/`rush build`. We would not need any special command. Only for the package release, we use the `autorest` command. (In most cases, it would be sufficient to say `autorest --typescript swagger/readme.md`. But there are several packages that need additional params)

Reply:
In that case, can we move it to the end? Am thinking the most commonly applicable sections should appear first

Reply:
Done